tests/unit/resource/CMakeLists.txt

```diff
@@ -22,6 +23,7 @@ set(tests
 
 set(cross_pool_injection_PARAMETERS THREADS_PER_LOCALITY -1)
 set(scheduler_priority_check_PARAMETERS THREADS_PER_LOCALITY -1)
+set(scheduler_binding_check_PARAMETERS THREADS_PER_LOCALITY -1)
```

Comment:
This needs to be enabled only if the shared priority queue scheduler is enabled.